verifiserte_termer.csv

```diff
@@ -420,7 +420,7 @@ logistisk funksjon,logistisk funksjon,logistic function,
 separasjon,separasjon,separation,
 venstrekoherent,venstrekoherent,left coherent,
 integrasjon,integrasjon,integration,
-pomengde,pomengd,poset,"Pomengde er en forkortelse for delvis ordnet mengde (bokmål) og delvis ordna mengd (nynorsk), og stammer fra <i>partially ordered set</i>."
+pomengde,pomengd,poset,"Genus: hankjønn<br><br>Pomengde er en forkortelse for delvis ordnet mengde (bokmål) og delvis ordna mengd (nynorsk), og stammer fra <i>partially ordered set</i>."
```

Comment:
Her synes jeg kanskje ikke vi trenger genus, siden det er ganske opplagt. Foreslår at vi kun har genus for forkortelser der det ikke inngår "ord".

Reply:
Enig